kedro/framework/cli/starters.py

```diff
@@ -280,6 +280,7 @@ def _get_cookiecutter_dir(
             no_input=True,
             directory=directory,
         )
+        print("****", template_path, cookiecutter_dir)
```

Comment:
I guess this isn't meant to be here any more...

Reply:
Indeed 😅